domain/src/main/java/com/example/domain/useCase/ApolloGetCountryUseCase.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.domain.useCase
+
+import com.example.domain.repository.ApolloCountryRepository
+
+class ApolloGetCountryUseCase(private val apolloCountryRepository: ApolloCountryRepository) {
```

Comment:
Again, let's rename this file and class to `GetCountryUseCase` only as tomorrow, we might not be using the `Apollo` client and then the name would be misleading.